adhocteam/Head-Start-TTADP
# IE keyboard navigation fix

**Description of change**
Redirects focus to the date input field after selecting a date from the calendar. This allows the keyboard navigation to continue in IE. 

**How to test**
- pull changes
- create a report in IE
- select a date
- use keyboard navigation
- verify that navigation moves forward and doesn't skip to the top

Alternatively use Chrome to test:
- pull changes
- create a report
- select a date
- verify that after date selection, the input date has focus

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/179

**Checklist**

- [n/a] Documentation updated
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
